src/extensions/codemirror/CodeMirror.tsx

```diff
@@ -280,6 +354,7 @@ export const CodeEditor = ({
             id={id ? id : name ? `codemirror-${name}` : undefined}
             ref={parent}
             data-test-id="codemirror-wrapper"
```

Comment:
`"codemirror-wrapper"` should only be used if `data-test-id` is not given. Please add also a `// @deprecated (v25) fallback with static test id will be removed` comment here.